docs/source/reference/changelog.md

```diff
@@ -8,7 +8,9 @@ command line for details.
 
 ## [Unreleased]
 
-### 4.0 (beta) - 2023-XX-YY
+## 4.0
+
+### 4.0.0 - 2023-04-20
```

Comment:
We missed out on this part:

> 4.0 should require no additional action beyond running upgrade-db (i.e. no action at all on z2jh with sqlite). It's otherwise a regular jupyterhub restart. That should be added to the notes!

Reply:
I figure I'll make a note of it in the z2jh changelog directly

Reply:
Good catch, sorry for missing that.

I think it's a safe bet that we'll make a 4.0.1 pretty soon with small fixes now that it's public.

Let's make sure to put it in announcements, etc. for now.